### DISCUSSION TITLE
Ability to download datasource config as yaml

### DISCUSSION THREAD

**Discussion by attachmentgenie** [grafana/grafana#56828] [Data Sources]
I know there are docs available on what flags are available to configure data sources as code, but lazy me would just love to configure a datasource by hand (to manually try what toggle/setting does what and or is required for my use case) and then just download it and add it to our IaC repos and manage the datasources from there from that point onwards

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
